core/templates/domain/objects/UnitsObjectFactory.ts

```diff
@@ -114,7 +119,9 @@ export class UnitsObjectFactory {
         }
       } else if (unitList[ind] === ')') {
         var elem = parenthesisStack.pop();
-        multiplier = parseInt(elem[1]) * multiplier;
+        if (typeof elem !== 'undefined') {
+          multiplier = elem[1] * multiplier;
+        }
```

Comment:
Why is this needed?

Reply:
Running the strict type checks with the original code gave the following errors:

core/templates/domain/objects/UnitsObjectFactory.ts(117,31): error TS2532: Object is possibly 'undefined'.

core/templates/domain/objects/UnitsObjectFactory.ts(117,31): error TS2345: Argument of type 'string | number' is not assignable to parameter of type 'string'.

Type 'number' is not assignable to type 'string'.